src/Qowaiv.CodeAnalysis.CSharp/Rule.cs

```diff
@@ -52,28 +52,36 @@ public static class Rule
             "Designing a class to support inheritance however, is hard. As a " +
             "consequence, it is considered a bad practice to unintentionally " +
             "allowing a class to be inheritable.",
-       category: Category.Design,
-       tags: new[] { "Design" });
+        category: Category.Design,
+        tags: new[] { "Design" });
 
     public static DiagnosticDescriptor OnlyUnsealedConcreteClassesCanBeInheritable => New(
         id: 0006,
         title: "Only unsealed concrete classes should be decorated as inheritable",
         message: "Remove the [{0}] attribute.",
         description:
             "The inheritable attribute is only meant to be used on concrete classes.",
-       category: Category.Design,
-       tags: new[] { "Design" });
+        category: Category.Design,
+        tags: new[] { "Design" });
 
     public static DiagnosticDescriptor UseFileScopedNamespaceDeclarations => New(
-       id: 0007,
-       title: "Use file-scoped namespace declarations",
-       message: "Use a file-scoped namespace declaration instead.",
-         description:
+        id: 0007,
+        title: "Use file-scoped namespace declarations",
+        message: "Use a file-scoped namespace declaration instead.",
+        description:
             "It reduces the number of braces in a file, and also eliminates the " +
             "wasted horizontal space to the left of the class definition, since " +
             "it no longer needs to be indented.",
-      category: Category.Design,
-      tags: new[] { "Design" });
+        category: Category.Design,
+        tags: new[] { "Design" });
+
+    public static DiagnosticDescriptor UseEmptyInsteadOfNullable => New(
+        id: 0008,
+        title: "Use empty value of type instead of ",
```

Comment:
Does something get added to the end of the title? (Or is it incomplete.) 

Also it suggests 'Use empty value', but for example in the test a compliant usage `public Guid Id { get; } // Compliant` does explicitly use an empty value. Is the idea here that the empty value is always the default?

Reply:
@renzo-baasdam That still has to be added. The thing is, I'm not happy het with the naming/documentation.

In an offline discussion with @Sjaaky we came to an improvement:
> Define the property as not-nullable as its type has an empty state.